Tests/StencilSwiftKitTests/StringFiltersTests.swift

```diff
@@ -140,11 +140,12 @@ extension StringFiltersTests {
 
   func testRemoveNewlines_WithTrue() throws {
     let expectations = [
-      "test": "test",
-      "  \n test": "test",
-      "test  \n ": "test",
-      "test\n \ntest": "testtest",
-      "\n test\n \ntest \n ": "testtest"
+      "test1": "test1",
+      "  \n test2": "test2",
+      "test3  \n ": "test3",
+      "test4\n \ntest": "test4test",
+      "\n test5\n \ntest test \n ": "test5test test",
+      "test6\ntest": "test6test"
```

Comment:
Maybe add that `" test test ":"test test"` here to confirm my suggestion about that it should remove the leading/trailing whitespaces in the examples you put in the md docs above